src/components/home/components/styles.css

```diff
@@ -79,6 +79,18 @@
 /*
  this is for Navibar.js and home.js
   */
+
+  .avatar-dropdown {
+     background-color: #f56a00;
+     font-size: 80;
+     margin-left: 630px;
```

Comment:
you can not set `margin-left 630px` it interrupts dynamic flow. Use `float: left` instead

Reply:
float-right did not work